src/bundle/Resources/config/default_parameters.yaml

```diff
@@ -86,3 +86,6 @@ parameters:
             name: keywords
             contentTypeIdentifier: image
             fieldDefinitionIdentifier: tags
+
+    #Used only for standalone DAM applications to limit content tree to Media -> Images folder only.
+    ibexa.dam_widget.image.images_folder_location_id: 51
```

Comment:
If it is used only for standalone DAMs, then should it be here on in dam related things (recipies?)

Reply:
I will add dedicated ConfigProvider to DAM project and there I will add rootLocationId parameter for standalone application.